blueoil/templates/lmnet/classification.tpl.py

```diff
@@ -85,6 +85,7 @@
 }
 NETWORK.WEIGHT_QUANTIZER = binary_mean_scaling_quantizer
 NETWORK.WEIGHT_QUANTIZER_KWARGS = {}
+NETWORK.QUANTIZE_FIRST_CONVOLUTION = {{ quantize_first_convolution }}
```

Comment:
Oh. I realized the quantized first convolution flag is only used in object detection. I'll omit this flag in other tasks.